server/controllers/banner.js

```diff
@@ -82,9 +86,9 @@ module.exports = {
       request({url: imageUrl, encoding: null}, (e, r, data) => {
         const contentType = r.headers['content-type'];
 
-        const imageHeight = 64;
+        const imageHeight = Math.round(maxHeight / 2);
```

Comment:
I'm not following the logic. Why are we dividing by 2, instead of setting the right number up above?